tools/autograd/derivatives.yaml

```diff
@@ -900,7 +900,7 @@
   self: zeros_like(grad, at::MemoryFormat::Preserve)
 
 - name: rsqrt(Tensor self) -> Tensor
-  self: -0.5 * grad * result.pow(3)
+  self: -0.5 * grad * result.pow(3).conj()
```

Comment:
currently gradgradcheck for `rsqrt` fails since it uses `pow_backward`. Will update `pow_backward` in follow up PRs and also add more extensive testing for it.